codegen-on-oss/codegen_on_oss/snapshot/context_snapshot.py

```
@@ -0,0 +1,205 @@
+        """
+        List all snapshots for a repository.
+
+        Args:
+            repo_name: Optional repository name to filter by
+
+        Returns:
+            A list of snapshot metadata
+        """
+        bucket_store = BucketStore(self.bucket_name)
+        s3_client = bucket_store.s3_client
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)